src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -262,14 +263,9 @@ export class InstructorCoursesPageComponent implements OnInit {
   }
 
   /**
-   * Creates a copy of a course including all sessions.
+   * Creates a copy of a course including the selected sessions.
    */
   onCopy(courseId: string, courseName: string, timeZone: string): void {
-    if (!courseId) {
-      this.statusMessageService.showErrorToast(`Course ${courseId} is not found!`);
-      return;
-    }
-
```

Comment:
Sorry regarding this: do you think there should still be a defensive mechanism in place?
Was commenting on this as the interpolated `courseId` is most likely redundant, but I wasn't rejecting the possibility of such error happening.
Can think of other places that might reuse this function.

Reply:
I see. I agree it will be better to have defensive mechanism